python/ray/tempfile_services.py

```diff
@@ -28,7 +29,13 @@ def make_inc_temp(suffix="", prefix="", directory_name="/tmp/ray"):
         "{directory_name}/{prefix}.{unique_index}{suffix}"
     """
     index = _incremental_dict[suffix, prefix, directory_name]
-    for seq in range(tempfile.TMP_MAX):
+    # `tempfile.TMP_MAX` could be extremely large,
+    # so we should use xrange in Python2.x
+    if sys.version_info[0] < 3:
```

Comment:
Instead of using `range` and `xrange`, can we just do `while index < tempfile.TMP_MAX`?